### ISSUE TITLE
Build bug

### ISSUE THREAD

**Issue by tamama** [ethereum/cpp-ethereum#4904]
Greetings,

I would like to report a build bug.
+ I attach the log as follows.
+ System: CentOS/7.4 , GCC-4.8.5 , CMake-3.6.3 , Ninja-1.7.2
+ Git commit : a24880812b7be8ceb96fa51cfad050c09fd95e97 
+ Git branch : develop

Kind regards,
Tamama

=====
LOG
======
```
[34/236] Performing install step for 'binaryen'
[0/1] Install the project...
-- Install configuration: "Release"
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/lib64/libbinaryen.a
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/include/binaryen-c.h
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/share/binaryen/wasm.js
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/share/binaryen/binaryen.js
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm-shell
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm-opt
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm-merge
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm-metadce
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/asm2wasm
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm2asm
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/s2wasm
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm-as
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm-dis
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm-ctor-eval
-- Installing: /root/cpp-ethereum/cpp-ethereum-build/deps/bin/wasm-reduce
[236/236] Linking CXX executable test/testeth
FAILED: test/testeth 
: && /bin/c++   -O2 -g  -fuse-ld=gold test/CMakeFiles/testeth.dir/tools/fuzzTesting/BoostRandomCode.cpp.o test/CMakeFiles/testeth.dir/tools/fuzzTesting/createRandomTest.cpp.o test/CMakeFiles/testeth.dir/tools/fuzzTesting/fuzzHelper.cpp.o test/CMakeFiles/testeth.dir/tools/jsontests/BlockChainTests.cpp.o test/CMakeFiles/testeth.dir/tools/jsontests/StateTests.cpp.o test/CMakeFiles/testeth.dir/tools/jsontests/TransactionTests.cpp.o test/CMakeFiles/testeth.dir/tools/jsontests/vm.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/BlockChainHelper.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/FillJsonFunctions.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/ImportTest.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/Options.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/Stats.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/TestHelper.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/TestOutputHelper.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/TestSuite.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/TestUtils.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/boostTest.cpp.o test/CMakeFiles/testeth.dir/tools/libtesteth/wast2wasm.cpp.o test/CMakeFiles/testeth.dir/tools/libtestutils/BlockChainLoader.cpp.o test/CMakeFiles/testeth.dir/tools/libtestutils/Common.cpp.o test/CMakeFiles/testeth.dir/tools/libtestutils/FixedClient.cpp.o test/CMakeFiles/testeth.dir/unittests/external-dependencies/boost.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcore/CommonJS.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcore/FixedHash.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcore/RangeMask.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcore/core.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcore/rlp.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcrypto/AES.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcrypto/LibSnark.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcrypto/MemTrie.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcrypto/SecretStore.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcrypto/crypto.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcrypto/hexPrefix.cpp.o test/CMakeFiles/testeth.dir/unittests/libdevcrypto/trie.cpp.o test/CMakeFiles/testeth.dir/unittests/libethashseal/EthashTest.cpp.o test/CMakeFiles/testeth.dir/unittests/libethcore/PrecompiledTest.cpp.o test/CMakeFiles/testeth.dir/unittests/libethcore/SealEngineTest.cpp.o test/CMakeFiles/testeth.dir/unittests/libethcore/commonjs.cpp.o test/CMakeFiles/testeth.dir/unittests/libethcore/dagger.cpp.o test/CMakeFiles/testeth.dir/unittests/libethcore/difficulty.cpp.o test/CMakeFiles/testeth.dir/unittests/libethcore/keymanager.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/Block.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/BlockChain.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/BlockChainInsert.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/BlockQueue.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/ClientBase.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/EthereumPeerTest.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/ExtVMTest.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/GasPricer.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/Genesis.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/SnapshotImporterTest.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/StateUnitTests.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/Transaction.cpp.o test/CMakeFiles/testeth.dir/unittests/libethereum/TransactionQueue.cpp.o test/CMakeFiles/testeth.dir/unittests/libp2p/capability.cpp.o test/CMakeFiles/testeth.dir/unittests/libp2p/eip-8.cpp.o test/CMakeFiles/testeth.dir/unittests/libp2p/net.cpp.o test/CMakeFiles/testeth.dir/unittests/libp2p/peer.cpp.o test/CMakeFiles/testeth.dir/unittests/libp2p/rlpx.cpp.o test/CMakeFiles/testeth.dir/unittests/libtesteth/blockchainTest.cpp.o test/CMakeFiles/testeth.dir/unittests/libtesteth/testHelperTest.cpp.o test/CMakeFiles/testeth.dir/unittests/libweb3core/memorydb.cpp.o test/CMakeFiles/testeth.dir/unittests/libweb3core/overlaydb.cpp.o test/CMakeFiles/testeth.dir/unittests/libweb3jsonrpc/AccountHolder.cpp.o test/CMakeFiles/testeth.dir/unittests/libweb3jsonrpc/Client.cpp.o test/CMakeFiles/testeth.dir/unittests/libweb3jsonrpc/jsonrpc.cpp.o  -o test/testeth  libethereum/libethereum.a libethashseal/libethashseal.a libweb3jsonrpc/libweb3jsonrpc.a libdevcrypto/libdevcrypto.a libdevcore/libdevcore.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib64/libcryptopp.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libyaml-cpp.a deps/lib/libbinaryen.a libethashseal/libethashseal.a libethash/libethash.a libwebthree/libwebthree.a libethereum/libethereum.a libevm/libevm.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libboost_program_options-mt.a libethcore/libethcore.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libsnappy.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libboost_fiber-mt.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libboost_context-mt.a libp2p/libp2p.a libdevcrypto/libdevcrypto.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib64/libcryptopp.a deps/lib/libsecp256k1.a deps/lib/libff.a deps/lib64/libmpir.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib64/libscrypt.a libdevcore/libdevcore.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libboost_filesystem-mt.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libboost_system-mt.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libboost_thread-mt.a -lpthread /usr/lib64/libleveldb.so /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libjsonrpccpp-server.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib/libjsonrpccpp-common.a /root/.hunter/_Base/8f7cf6d/7cd4862/8b65c20/Install/lib64/libjsoncpp.a deps/src/binaryen-build/lib/libwasm.a deps/src/binaryen-build/lib/libasmjs.a deps/src/binaryen-build/lib/libpasses.a deps/src/binaryen-build/lib/libcfg.a deps/src/binaryen-build/lib/libir.a deps/src/binaryen-build/lib/libemscripten-optimizer.a deps/src/binaryen-build/lib/libsupport.a && :
c++: error: deps/lib/libbinaryen.a: No such file or directory
ninja: build stopped: subcommand failed.
[root@london-node-224 cpp-ethereum-build]# 
```

**Comment by axic** [comment]
@chfast was the hera submodule updated after the `lib64` fix?

**Comment by chfast** [comment]
Not sure will, check.
Can I update it to the recent master?

**Comment by tamama** [comment]
@chfast @axic 
Fixed in commit aea3bc120761fe27fc696d0649ba5ae7a56a7915
(CentOS/7.4 , GCC-4.8.5 , CMake-3.6.3 , Ninja-1.7.2)
